docker-compose.yml

```diff
@@ -77,20 +77,65 @@ services:
       temporal-admin-tools:
         condition: service_healthy
 
-  flow_worker:
-    container_name: flow_worker
+  flow_worker1:
```

Comment:
create a new file - `multi-worker.docker-compose.yml`, leave the current one as minimal.